pom.xml

```diff
@@ -72,6 +72,14 @@
             <version>2.8.0</version>
         </dependency>
 
+        <!-- TODO: Add the findbugs plugin and use findbugs annotations more
```

Comment:
We are. It's the source of `@ThreadSafe`. The TODO is to add the maven plugin that scans the annotations provided by this dependency.